tiup/tiup-cluster.md

```diff
@@ -543,3 +543,43 @@ etcdctl [args] = tiup ctl etcd [args]
 ```bash
 tiup ctl pd -u http://127.0.0.1:2379 store
 ```
+
+## 部署机环境检查
+
+使用 `check` 子命令可以对部署机的环境进行一系列检查，并输出检查结果。通过执行 `check` 子命令，可以发现常见的不合理配置或不支持情况。命令参数列表如下：
+
+```bash
+Usage:
+  tiup cluster check <topology.yml | cluster-name> [flags]
+
+Flags:
+      --apply                  Try to fix failed checks
+      --cluster                Check existing cluster, the input is a cluster name.
+      --enable-cpu             Enable CPU thread count check
+      --enable-disk            Enable disk IO (fio) check
+      --enable-mem             Enable memory size check
+  -h, --help                   help for check
+  -i, --identity_file string   The path of the SSH identity file. If specified, public key authentication will be used.
+  -p, --password               Use password of target hosts. If specified, password authentication will be used.
+      --user string            The user name to login via SSH. The user must has root (or sudo) privilege.
+```
+
+默认情况下，此功能用于在部署前进行环境检查，通过指定 `--cluster` 参数切换模式，也可以用于对已部署集群的部署机进行检查，例如：
+
+```bash
+# check deploy servers before deploy
+tiup cluster check topology.yml --user tidb -p
+
+# check deploy servers of an existing cluster
+tiup cluster check <cluster-name> --cluster
```

Comment:
Prefer `tiup cluster check  --cluster <cluster-name>`

Reply:
Both will work, I use this style because our usage string shows `cluster <topology.yaml | cluster-name> [flags]`